src/Tokenizer/Analyzer/AlternativeSyntaxAnalyzer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer;
+
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @internal
```

Comment:
could you add a `@TODO` that on 4.0 we remove this analyzer and move this logic into a transformer to make the `:` a custom token?
(TBH I forgot to add that to the `GotoLabelAnalyzer`, we should move that to a transformer as well)